package.json

```diff
@@ -43,5 +43,8 @@
   "engines": {
     "node": ">=16"
   },
+  "peerDependencies": {
+    "@blockfrost/openapi": "0.1.44-beta"
```

Comment:
would something like `>= 0.1.44` work for us?, otherwise we would need to update this line with every release to prevent warnings, which is not feasible.

Reply:
alternatively we could left fs read for the users and only pass the content of the file as a parameter to `getSchemaForEndpoint`